gemd/entity/value/smiles_value.py

```diff
@@ -0,0 +1,35 @@
+"""An empirical chemical formaula."""
+from gemd.entity.value.molecular_value import MolecularValue
+
+
+class SMILES(MolecularValue):
```

Comment:
@djack What is Citrine Best Practice for acronyms in class names?  Should this be `SMILES`, `Smiles` or other?

Reply:
It's `Smiles` in the backend, fwiw.